components/script_traits/lib.rs

```diff
@@ -796,6 +797,10 @@ pub enum ConstellationMsg {
     CloseBrowser(TopLevelBrowsingContextId),
     /// Make browser visible.
     SelectBrowser(TopLevelBrowsingContextId),
+    /// Forward an event to the script task of the given pipeline.
+    ForwardEvent(PipelineId, CompositorEvent),
+    /// Requests that the constellation inform the compositor of the a cursor change.
```

Comment:
This comment doesn't make sense, since the compositor is the one that sends this message.